modules/exploits/windows/smb/ms17_010_eternalblue.rb

```diff
@@ -286,14 +295,15 @@ def print_core_buffer(os)
     end
   end
 
+  '''
```

Comment:
Apparently this works in Ruby, not just Python, lol. :+1:

Reply:
Is there any reason to leave this block here?

Reply:
Move the comment and delete the code, IMHO.

Reply:
It seemed odd enough, and I didn't know the reasoning behind it, that I wasn't comfortable removing it outright without a couple other sets of eyes.  In my testing, it seemed unnecessary.

If you guys are cool with taking it out, I'm happy to do so.